src/components/ProjectCard/ProjectCard.tsx

```diff
@@ -1,33 +1,42 @@
 import * as React from 'react'
+import { Link } from 'react-router-dom'
 
+import { locations } from 'routing/locations'
 import { Props } from './ProjectCard.types'
 import { getProjectDimensions } from 'modules/project/utils'
 import './ProjectCard.css'
 
 export default class ProjectCard extends React.PureComponent<Props> {
-  static defaultProps = {
-    onClick: (_: any) => {
-      /*noop */
-    }
-  }
-
   handleOnClick = () => {
-    this.props.onClick(this.props.project)
+    const { onClick } = this.props
+    if (onClick) {
+      onClick(this.props.project)
+    }
   }
 
   render() {
-    const { project } = this.props
+    const { project, onClick } = this.props
 
-    return (
-      <div className="ProjectCard Card" onClick={this.handleOnClick}>
-        {/*<img src={project.thumbnail} alt={project.title} width={248} height={184} />*/}
+    // <img src={project.thumbnail} alt={project.title} width={248} height={184} />
```

Comment:
Why is this commented?

Reply:
It was there just as a reminder that we needed to add a thumbnail in the future, but I can remove it!

Reply:
cool, leave it 👍